src/views/health.html

```diff
@@ -0,0 +1,7 @@
+<html>
```

Comment:
I feel like we should add a `<!doctype html>` tag and a `<head>` section with a `<title>` as a bare minimum to this view.

Reply:
Yes fair enough, I was trying to keep it to the bare minimum. The component default is just to return a 200 reponse with an 'i'm healthy' textual response but if we are returning HTML then agreed that it should include the tags you have identified.

Reply:
👍